tests/NRediSearch.Test/QueryTest.cs

```diff
@@ -50,9 +52,15 @@ public void SerializeRedisArgs()
             Assert.Contains("WITHSCORES".Literal(), args);
             Assert.Contains("LANGUAGE".Literal(), args);
             Assert.Contains("", args);
+            Assert.Contains("SCORER".Literal(), args);
+            Assert.Contains("TFIDF.DOCNORM", args);
+            Assert.Contains("EXPLAINSCORE".Literal(), args);
```

Comment:
@jjchiw I think you should also add a test that runs a Search with EXPLAINSCORE and validate the result

Reply:
Done :)
Here: tests/NRediSearch.Test/ExampleUsage.cs
https://github.com/StackExchange/StackExchange.Redis/pull/1254/files#diff-5f7cf341c5a99b172f218d160f44864dR79